specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/table.json

```diff
@@ -0,0 +1,526 @@
+            "schema": {
+              "$ref": "#/definitions/ListTableResource"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ListTableServices": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TableServiceProperties"
+          },
+          "description": "List of table services returned.",
+          "readOnly": true
+        }
+      }
+    },
+    "TableServiceProperties": {
+      "properties": {
+        "properties": {
+          "properties": {
+            "cors": {
+              "$ref": "./common.json#/definitions/CorsRules",
+              "description": "Specifies CORS rules for the Table service. You can include up to five CorsRule elements in the request. If no CorsRule elements are included in the request body, all CORS rules will be deleted, and CORS will be disabled for the Table service."
+            }
+          },
+          "x-ms-client-flatten": true,
+          "x-ms-client-name": "TableServiceProperties",
+          "description": "The properties of a storage account’s Table service."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/Resource"
+        }
+      ],
+      "description": "The properties of a storage account’s Table service."
+    },
+    "Table": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/TableProperties",
+          "x-ms-client-flatten": true,
+          "x-ms-client-name": "TableProperties",
+          "description": "Table resource properties."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/Resource"
+        }
+      ],
+      "description": "Properties of the table, including Id, resource name, resource type."
+    },
+    "TableProperties": {
+      "properties": {
+        "tableName": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Table name under the specified account"
+        }
+      }
+    },
+    "ListTableResource": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Table"
+          },
+          "description": "List of tables returned.",
+          "readOnly": true
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Request URL that can be used to query next page of tables",
+          "readOnly": true
+        }
+      },
+      "description": "Response schema. Contains list of tables returned"
+    },
+    "CloudError": {
```

Comment:
As CloudError are defined same in File/Queue/Table service 3 times, we might should move it to common.json

Reply:
Yes, I will make a separate PR for this purpose.